buildscripts/ci/windows/deploy.bat

```diff
@@ -0,0 +1,12 @@
+REM @echo off
```

Comment:
Would you like to write them on cmake? - it's quite simple, universal and cross-platform 
like this https://github.com/musescore/MuseScore/blob/master/buildscripts/ci/crashdumps/ci_generate_and_upload.cmake

Reply:
This is not really cross platform though (`windeployqt` is OS-specific). I don't see much benefit in using CMake here; shell script seems a more natural choice. Using CMake feels like abusing it for something that is not its purpose, so we should not do that unless there is a very good reason for it.

Reply:
Yes, I understand that this is not quite a cross-platform script...
I'm rather saying that using cmake, we can make scripts uniform, designed the same way, with the same practices...
If cmake allows us to run in script mode, then it is a fully scripting language, and quite convenient :)